### PR TITLE
Pagination & larger collection sizes

### PR DESCRIPTION
**Pull request by cohesiveneal**
Webflow updated their API to allow for limit & offset parameters on the /collection/ID/items API call. This commit adds a #paginate_items method and refactors Client#items to fetch larger sets (using the #paginate_items method).

### CODE REVIEW

File: lib/webflow/client.rb [hunk 1]
```diff
@@ -48,9 +48,37 @@ def collection(collection_id)
...
+    def paginate_items(collection_id, per_page: 100, page: 1)
+      get("/collections/#{collection_id}/items", params: { limit: per_page, offset: per_page * (page - 1) })
+    end
+
+
     def items(collection_id, limit: 100)
-      json = get("/collections/#{collection_id}/items", params: {limit: limit})
-      json['items']
+      fetched_items = []
+      num_pages     = (limit.to_f / 100.0).ceil
```
Comment (phoet) [inline lib/webflow/client.rb]: do you know `number.fdiv` ?

File: lib/webflow/client.rb [hunk 2]
```diff
@@ -48,9 +48,37 @@ def collection(collection_id)
...
+      per_page      = limit > 100 ? 100 : limit
+
+      num_pages.times do |i|
```
Comment (phoet) [inline lib/webflow/client.rb]: i like this pattern to avoid a while loop 👍

**Comment by phoet** [comment]
thank you @cohesiveneal ! would you be interested in becoming a maintainer? it's always nice to more people who are able to contribute.

**Comment by cohesiveneal** [comment]
hey @phoet - sure thing, happy to! thanks!

**Comment by phoet** [comment]
@cohesiveneal hello and welcome dear collaborator! feel free to merge this PR yourself.

**Pull request merged**